pkg/utils/utils_virtual_test.go

```diff
@@ -0,0 +1,44 @@
+package utils
+
+import (
+	"testing"
+
+	"github.com/jaypipes/ghw"
+	"github.com/jaypipes/ghw/pkg/net"
+	"github.com/jaypipes/ghw/pkg/option"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/utils/pointer"
+)
+
```

Comment:
can you make this one a ginkgo test ?

like we have for daemon package (daemon_test.go)